### ISSUE TITLE
Invert Y-axis option for navigation in Editor 3d-view

### ISSUE THREAD

**Issue by datgame** [godotengine/godot#16926]
Please add an option for inverting the y-axis of mouse movements when navigating the 3d-world during editing.
I know Unity and UE don't have this simple feature, but we can do better.
I see no way to add the feature request label myself here on github.

**Comment by akien-mga** [comment]
Could you be more specific? There's an issue template for a reason.

**Comment by datgame** [comment]
I had read the template, and it provided no help. Not the info for first-time posters either.

**Comment by Zylann** [comment]
Do you want this too for orbit navigation?

**Comment by datgame** [comment]
yes, for all kinds of navigation, both look up, and when rotating things.
possibly some people want invert y-axis on/off for different situations.  i heard some people even use invert x-axis, but that's very rare.

**Comment by Noshyaar** [comment]
>I had read the template, and it provided no help. Not the info for first-time posters either.

All issue filers are required to read https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md first.

This is the template:
```
<!-- Please search existing issues for potential duplicates before filing yours:
https://github.com/godotengine/godot/issues?q=is%3Aissue
-->

**Godot version:**
<!-- Specify commit hash if non-official. -->

**OS/device including version:**
<!-- Specify GPU model and drivers if graphics-related. -->

**Issue description:**
<!-- What happened, and what was expected. -->

**Steps to reproduce:**

**Minimal reproduction project:**
<!-- Recommended as it greatly speeds up debugging. Drag and drop a zip archive to upload it. -->
```
Apparently you noticed it and deleted it without trying to fill in the blanks first.

I'll break it down in sections:
```
**Godot version:**
```
Here's where you put the editor version where the feature applies. I could assume that you want this feature in Godot 2.2

```
**OS/device including version:**
```
Probably not related in this case, but some features work in one platform but not others. Would be nice to know. Your proposed feature could already be implemented, but not usable in your platform.

```
**Issue description:**
<!-- What happened, and what was expected. -->
```
What happened or didn't happen that makes you think of this feature. Some features are unnecessary to everyone's workflow. This gives us backgrounds to support if it's worth it to implement it for everyone. Every features take time.

```
**Steps to reproduce:**
**Minimal reproduction project:**
```
For bug reports.

If the template provides no help, I'd say that's another issue. Still, ignore the template and your issue might be overlooked.

**Comment by DoctorWhoof** [comment]
New user here. This is literally the very first thing I ran into when trying Godot: I'm using it on a laptop with trackpad, and BOTH axis should have the option to invert controls.

Orbit and pan are inverted - it should drag "along" the two finger gesture on the trackpad, not against it - but Zoom is actually correct, so the inverted controls should be per action?

Switching to Maya mode works as expected (movement along gesture, not against it)... but it requires a three button mouse, and doesn't work with trackpad gestures at all.

Thanks

**Comment by Zireael07** [comment]
There is an option to emulate a three button mouse, I use Maya mode with a trackpad.

**Comment by datgame** [comment]
The option is now there in settings -> editors -> 3d -> invert y-axis, but not on in 3.0.6 on steam.
The change was made 8 months ago, so I don't know why it's not in recent releases. 
https://github.com/godotengine/godot/blame/a3501c096f32506bf2262e36a302bf65f4ad0f4a/editor/plugins/spatial_editor_plugin.cpp#L1904
I built it locally, and it works!

**Comment by akien-mga** [comment]
Closing as fixed by #17505.

> The option is now there in settings -> editors -> 3d -> invert y-axis, but not on in 3.0.6 on steam.
The change was made 8 months ago, so I don't know why it's not in recent releases.

New developments are done in the master branch, which is where the next minor release (3.x) is being developed, currently 3.1. This will therefore be available on Steam when 3.1-stable is released.

The PR is marked as `cherrypick`, so it might be considered to be cherry-picked to the current stable branch (3.0.x) when our stable branch maintainers get to it.

**Comment by molevision** [comment]
The invert Y axis option also inverts mouse movement (such as when fly navigating while holding down right click). This is unexpected behaviour, and the invert X axis option does not do this. Ideally neither should invert Y.
